Automattic/vip-go-mu-plugins
# Disable JP Search when using VIP ES

## Description

Auto disable JP Search when VIP ES is active, to prevent conflicts.

Branched from #1394 (to prevent conflicts) so that should be merged first.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Enable JP Search on a site via UI
1. Set `USE_VIP_ELASTICSEARCH` to `true`
1. Ensure JP Search is now disabled
1. Enable JP Search [via code](https://wpvip.com/documentation/vip-go/elasticsearch-on-vip-go/#how-to-activate-the-search-module)
1. Disable `USE_VIP_ELASTICSEARCH` to ensure above filter is working
1. Re-enable the constant, ensure JP Search is now disabled
